pipelines/broad/rna_seq/RNAWithUMIsPipeline.wdl

```diff
@@ -75,29 +74,18 @@ workflow RNAWithUMIsPipeline {
     population_vcf_index: "Population VCF index file used for contamination estimation"
   }
 
-  call tasks.VerifyPipelineInputs {
```

Comment:
You still should keep this task and have it verify that either the bam is defined OR a pair of fastqs are both defined.
You could do so here, in this WDL, or in the tasks wdl (but if you do it in the top-level, then the task should be deleted from the task wdl).

Reply:
@takutosato just making sure you saw this

Reply:
Actually, I just added it back in.